src/sentry/static/sentry/app/views/releases/detail/shared/releaseCommits.jsx

```diff
@@ -28,7 +29,8 @@ const ReleaseCommits = createReactClass({
   componentDidMount() {
     this.api.request(this.getPath(), {
       method: 'GET',
-      data: this.props.location.query,
+      // We need to omit global selection header url params because they are not supported
+      data: omit(this.props.location.query, Object.values(URL_PARAM)),
```

Comment:
I wonder if we should just omit the entire data object here an in release artifacts? We don't ever want to send anything right (i.e. we don't paginate this)?

Reply:
It looks like we do paginate: https://github.com/getsentry/sentry/pull/12158/files/4c02cdf3af4d493e5df09a62e74dd4de3644c127#diff-ffa7f78a6c66f3f694d98c40beaa7264R39